### ISSUE TITLE
A way to clear selection

### ISSUE THREAD

**Issue by isuke01** [sagalbot/vue-select#1029]
Hey,

I faced a issue where I cant clear selection easy way.
I have a situation where user can see events form past or feature, plus dropdown with categories, I'd like to clear selected categories when user switch view  form past to feature, and all the way around.

For now only way, that work for me is to select element with class vs_close and trigger click on them, but it does not seems to clean solution.
I tried clear v-mounted varible, also varible using value and @input set, nothing works :)

Is there any better solution to do it, in both single and multi select.

Thanks for any suggestion :D

**Comment by Ahmedjabar** [comment]
Hi @isuke01 , 

Clearing the selection worked for me using v-model.

```
<v-select
  required
  :options="countriestList"
  **v-model="form.country_id"**
  :reduce="country => country.key"
></v-select>
```
I had a reset method that sets `form.country_id` to null and thus clears the selection. Hope that helps.

**Comment by isuke01** [comment]
Looks like it works...

Yes sorry, it works in Pen. https://codepen.io/isuke01/pen/BayWYvV?editors=1010
I have created few use cases, and every time works. 
But for some reason it wont on my local, even with exact same code. 
I'll do more research what is different :)

Thanks!

**Comment by isuke01** [comment]
Ok Sorry for double post.
I have found what was the issue!

I have included this script from docs page 'https://unpkg.com/vue-select@3.0.0 ' 
I won't to use latest due to possibility changes in future, so that's why I picked this other one, I was sure it was the newest one :P 
And in this version it just don't work (it also expalins some other issues that I had) :D But now everything is fixed, I have selected `3.4.0` version now, and works like charm!

Thanks, and sorry for issue!
